darts/tests/test_global_forecasting_models.py

```diff
@@ -177,3 +178,60 @@ def test_same_result_with_different_n_jobs(self):
                 pred2 = model.predict(n=36, series=multiple_ts, n_jobs=-1)  # assuming > 1 core available in the machine
 
                 self.assertEqual(pred1, pred2, 'Model {} produces different predictions with different number of jobs')
+
+        @patch('darts.models.torch_forecasting_model.torch.save')
+        @patch('darts.models.torch_forecasting_model.TorchForecastingModel._train')
+        @patch('darts.models.torch_forecasting_model.shutil.rmtree')
+        def test_fit_with_constr_epochs(self, rmtree_patch, train_patch, save_patch):
```

Comment:
Are you using the `save_patch` anywhere in any of the tests?

Reply:
no, but it has to be there for the test to skip saving on disk during testing and go faster, but I'm not checking whether it is called. But since I'm using the `patch` I need to provide it as a param to the function.